config.toml

```diff
@@ -3,6 +3,7 @@ languageCode = "en-us"
 title = "Flynt Framework"
 metadataformat = "yaml"
 canonifyurls = true
+relativeURLs = true
```

Comment:
this leads to assets being referenced with `../../` rather than `https://docs.flynt.com/`